### PR TITLE
Add support for clang-format 12

### PR DESCRIPTION
**Pull request by alkino**
Clang 12 is out !

### CODE REVIEW

File: cpp/clang-format-12 [hunk 1]
```diff
@@ -0,0 +1,93 @@
+---
+AccessModifierOffset: -2
+AlignAfterOpenBracket: Align
+AlignConsecutiveAssignments: false
+AlignConsecutiveDeclarations: false
+AlignConsecutiveMacros: true
```
Comment (pramodk) [inline cpp/clang-format-12:6]: Just for clarity- is this same file as previous version or any modifications needed? and tested locally?

**Review by pramodk — APPROVED** [review]
LGTM

File: cpp/clang-format-12 (hunk 1 above)
Comment (alkino) [reply cpp/clang-format-12:6]: copy of clang-format-11, tested on nmodl

Comment (pramodk) [reply cpp/clang-format-12:6]: Ok

**Pull request merged**